src/datachain/sql/sqlite/types.py

```diff
@@ -28,26 +28,21 @@ def get_col_spec(self, **kwargs):
 
 
 def adapt_array(arr):
-    return orjson.dumps(arr).decode("utf-8")
+    return json.dumps(arr, ensure_ascii=False)
```

Comment:
May be also add `separators` here to keep json minified in db?

Reply:
Oh, great! Then remove `separators` in other places? 👀

Reply:
yes, they are redundant ... removing